src/controller/python/chip/yaml/runner.py

```diff
@@ -543,8 +598,40 @@ def decode(self, result: _ActionResult):
 
         return decoded_response
 
+    def _get_fabric_id(self, identity):
+        if identity == 'alpha':
+            return 1
```

Comment:
maybe we should have constants like `FABRIC_ID_ALPHA = 1` in case we ever reuse things somewhere.

also alternative enum based:

```python
from enum import Enum

class TestFabricId(Enum):
     ALPHA=1
     BETA=2
     GAMMA=3

# separate method may not be needed
def _get_fabric_id(id):
    return TestFabricId[id.upper()].value
```

not sure if this improves readability though. It may.